dev/src/Command/NewComponentCommand.php

```diff
@@ -64,6 +64,7 @@ class NewComponentCommand extends Command
     private $rootPath;
     private $httpClient;
     private RunProcess $runProcess;
+    private int $timeout;
```

Comment:
Since this is only used once in the `execute` function, I don't see a reason why we need to save it as a property... We can just consume it as a local variable.